readthedocs/projects/forms.py

```diff
@@ -444,23 +445,46 @@ def save(self):
 
 class WebHookForm(forms.ModelForm):
 
-    """Project webhook form."""
+    project = forms.CharField(widget=forms.HiddenInput(), required=False)
```

Comment:
I don't think we need this field here. The project should be grabbed from the inherent view instead. Otherwise, the user can modify the `project=` sent and add webhooks to a different project.

Reply:
Project isn't read from the user input, we always use the one given by the view

https://github.com/readthedocs/readthedocs.org/blob/c5312865666fd8bf76056cb7cf7e28d2446d01c1/readthedocs/projects/forms.py#L476-L477

this is so we don't need to override the save method from the form. We use this pattern in several forms in the same file.